include/superior_mysqlpp/low_level/dbdriver.hpp

```diff
@@ -331,11 +353,24 @@ namespace SuperiorMySqlpp { namespace LowLevel
         template<typename Iterable>
         std::string makeHexString(const Iterable& from)
         {
-            auto fromSize = std::end(from) - std::begin(from);
-            auto buffer = std::make_unique<char[]>(fromSize*2 + 1);
-            auto length = mysql_hex_string(buffer.get(), from, fromSize);
+            return makeHexString(from, std::end(from) - std::begin(from));
```

Comment:
Are you sure about if the first argument is correct? You call `std::string makeHexString(T *form, std::size_t formlen)` function, where first argument is pointer and you are currently passing reference.

Reply:
Well, it's expected that first argument will be convertible to pointer type (char[] for instance), in that case argument will be type of const reference to pointer.